src/visit.js

```diff
@@ -1,6 +1,8 @@
 const { caseOf } = require('./functional')
 const { RRule } = require('rrule')
 
+const decimal = val => val && parseInt(val, 10)
```

Comment:
👎 what is this supposed to do ? `parseInt` will return a `NaN` when a wrong value is supplied, here it will return a boolean.

Reply:
It does not return a boolean, it returns undefined or an int

Reply:
If you know where undefined values are removed from parsing, you can fix it otherwise, I will and it might be done in another PR

Reply:
Well, `parseInt` returns `NaN` or an int. How is this logical that the boolean operator `&&` returns `undefined | int` ?

Stop defensive programming, the behavior is already handled by `parseInt`.

Reply:
This is not defensive programming, this is a fix where values are decimals but converted to string. The tests without deprecated function shows that.

Reply:
I remove this code from this PR and make dedicated PR for change string to decimals when this is possible not for interval for example)